_docs/changing_dns.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: docs
+status: publish
+published: true
+title: Changing DNS
+permalink: /docs/changing_dns/
+author:
+  display_name: Even Becker
+  login: evenbecker
+  url: ''
+author_login: demm
+# Even Becker won't show his email address
+# Feel free to add authors and to modify this file
+#wordpress_id:
+#wordpress_url:
+date: '2024-05-01 10:06:56 +0200'
+date_gmt: '2024-05-01 10:06:56 +0200'
+categories: []
+tags: []
+comments: []
+---
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+**Changing/encrypting the DNS**
```

Comment:
Docs layout automatically adds the title:
https://github.com/KaOSx/kaosx.github.io/blob/master/_layouts/docs.html#L22

Don't repeat that here